test/lib/RuleManager.test.ts

```diff
@@ -601,7 +601,7 @@ describe('RuleManager', () => {
 				// Validate the results.
 				expect(results.length).to.equal(1, 'Wrong number of targets matched');
 				expect(results[0].target).to.equal(targets[0], 'Wrong glob matched');
-				expect(results[0].isDirectory).to.not.equal(true, 'Should not be flagged as directory');
+				expect(results[0].targetType).to.equal(TargetType.GLOB, 'Should be flagged as glob');
```

Comment:
526 is expecting a file because the target strings defined at line 507 are all paths to individual files. This line expects a glob because the targets defined at 589 are all globs.